src/data/lovelace.ts

```diff
@@ -18,6 +18,11 @@ export interface LovelaceViewConfig {
   theme?: string;
   panel?: boolean;
   background?: string;
+  show?: boolean | ShowViewConfig[];
```

Comment:
I don't like `show` . Show what? I think that it's more descriptive if we make it `show_if`

Reply:
Wonder if we should change it from `show` to `visible`.

`visible: false` and

```yaml
visible:
 - user: abcd
```

both clearly describe intention of the config option.

Reply:
Agreed, `visible` is much better/clearer